### PR TITLE
Book an appointment Front-end: Navigation

### PR DESCRIPTION
**Pull request by Debas-31**
- [x] The user to navigate through the following links: Solved https://github.com/sentayhu19/Book-an-appointment-back-end/issues/4
  - Motorcycles that you selected as a theme.
  - Reservation
  - My reservations
  - Add a motorcycle

### CODE REVIEW

**Comment by AyoubM490** [comment]
Please fix the linter errors

File: src/layout/navigation/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <div>
+        <Link to="/">
+          <h1 className="text-center text-dark"><i>Power Motors</i></h1>
+        </Link>
+      </div>
+      <div className="d-flex flex-column center-align">
+        <NavLink to="#" exact activeClassName="active" className="nav-link p-3">MOTORSCYCLES</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">RESERVATIONS</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">MY RESERVATION</NavLink>
+        <NavLink to="#" activeClassName="active" className="nav-link p-3">ADD MOTORCYCLE</NavLink>
```
Comment (Achilles-Dev) [inline src/layout/navigation/Sidebar.js]: Links on the nav.

- [ ] Motorcycles
- [ ]  "Reserve" form.
- [ ]  My reservations.
- [ ]  Add motorcycle
- [ ]  Delete motorcycle

Please make sure your navigation links have the above names.

**Review by Achilles-Dev — CHANGES_REQUESTED** [review]
Hi @Debas-31 :wave:

Good job implementing the design guidelines.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Review by sentayhu19 — APPROVED** [review]
Hi @Debas-31 -Dev
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_Check for optional comments below_

Cheers and Happy coding!👏👏👏

**Pull request merged**